### PR TITLE
Document object_dnq (Deku Princess)

### PR DESCRIPTION
**Pull request by tom-overton**
Also did some light doc on the actor to understand the context behind these assets.

### CODE REVIEW

File: src/overlays/actors/ovl_En_Dnp/z_en_dnp.c [hunk 1]
```diff
@@ -245,7 +282,11 @@ s32 func_80B3D044(EnDnp* this, PlayState* play) {
 
 void func_80B3D11C(EnDnp* this, PlayState* play) {
     static s32 D_80B3DE74[] = {
```
Comment (EllipticEllipsis) [inline src/overlays/actors/ovl_En_Dnp/z_en_dnp.c:284]: This doesn't seem that common a pattern, could just call it `sCsAnimations`

**Review by EllipticEllipsis — APPROVED** [review]

**Review by AngheloAlf — APPROVED** [review]

File: assets/xml/objects/object_dnq.xml [hunk 2]
```diff
@@ -1,88 +1,99 @@
 ﻿<Root>
     <File Name="object_dnq" Segment="6">
```
Comment (hensldm) [inline assets/xml/objects/object_dnq.xml:2]: File header comment please :)

File: assets/xml/objects/object_dnq.xml [hunk 3]
```diff
@@ -1,88 +1,99 @@
...
-        <Limb Name="object_dnq_Standardlimb_010CE4" Type="Standard" EnumName="OBJECT_DNQ_LIMB_18" Offset="0x10CE4" />
-        <Limb Name="object_dnq_Standardlimb_010CF0" Type="Standard" EnumName="OBJECT_DNQ_LIMB_19" Offset="0x10CF0" />
-        <Skeleton Name="object_dnq_Skel_010D60" Type="Flex" LimbType="Standard" LimbNone="OBJECT_DNQ_LIMB_NONE" LimbMax="OBJECT_DNQ_LIMB_MAX" EnumName="ObjectDnpLimb" Offset="0x10D60" />
-        <Animation Name="object_dnq_Anim_0115B8" Offset="0x115B8" />
-        <Animation Name="object_dnq_Anim_012428" Offset="0x12428" />
+        <!-- Deku Princess Animations -->
+        <Animation Name="gDekuPrincessJumpKickAnim" Offset="0x7D8" /> <!-- Original name is "dnq_atack" -->
+        <Animation Name="gDekuPrincessJumpToKingAnim" Offset="0x125C" /> <!-- Original name is "dnq_attack1" -->
+        <Animation Name="gDekuPrincessBounceOnKingStartAnim" Offset="0x17F8" /> <!-- Original name is "dnq_attack2" -->
+        <Animation Name="gDekuPrincessBounceOnKingLoopAnim" Offset="0x1C1C" /> <!-- Original name is "dnq_attack2_loop" -->
```
Comment (hensldm) [inline assets/xml/objects/object_dnq.xml:7]: Hmm, not 100% sure but I feel like we could possibly drop the `ToKing` and `OnKing` in these animations.

**Review by jpburnett — APPROVED** [review]
Noice

**Pull request merged**